x-pack/plugins/security/server/lib/authorization/actions/actions.ts

```diff
@@ -22,17 +22,17 @@ export class Actions {
    */
```

Comment:
Now that we're relying on the version number a bit more, how do you feel about adding a sanity check to make sure it's at least a non-empty string before it passes through to the underlying action classes? It shouldn't ever be a problem, but I'd hate to silently have `api:undefined:*` or `api::*`show up as a registered action

Reply:
That makes sense, I will make it so.